group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -271,18 +277,17 @@ public void testCreateAssignmentMemberSpec() {
             mkTopicAssignment(barTopicId, 1, 2, 3)
         ));
 
-        AssignmentMemberSpec assignmentMemberSpec = createAssignmentMemberSpec(
+        MemberSubscriptionSpec subscriptionSpec = createMemberSubscriptionSpecImpl(
             member,
             assignment,
             topicsImage
         );
 
-        assertEquals(new AssignmentMemberSpec(
-            Optional.of("instanceId"),
+        assertEquals(new MemberSubscriptionSpecImpl(
             Optional.of("rackId"),
             new TopicIds(mkSet("bar", "foo", "zar"), topicsImage),
-            assignment.partitions()
-        ), assignmentMemberSpec);
+            assignment
+            ), subscriptionSpec);
```

Comment:
nit: The indentation seems to be off here. I think that it should be as it was before.